DEPS.xwalk

```diff
@@ -47,6 +47,11 @@ solutions = [
       'src/third_party/khronos/CL':
         crosswalk_git + '/khronos-cl-api-1.2.git@6f4be98d10f03ce2b12c769cd9835c73a441c00f',
 
+      # Include Intel RSSDK headers and library loader for RealSense camera
+      # support.
+      'src/third_party/libpxc':
+        crosswalk_git + '/libpxc.git@WM5_1',
```

Comment:
I don't think there's a downside to using a tag here, but you might want to consider using a full hash just to follow what the other entries (including `DEPS` upstream) do.